wandb/apis/internal.py

```diff
@@ -10,6 +10,19 @@ def __init__(self, *args, **kwargs):
         self._api_kwargs = kwargs
         self._api = None
 
+    def __getstate__(self):
+        """Used for serializing. self._api is not serializable,
+        so it's dropped"""
+        state = self.__dict__.copy()
+        del state["_api"]
+        return state
+
+    def __setstate__(self, state):
+        """Used for deserializing. Don't need to set self._api because
```

Comment:
Mentioned elsewhere, but I think if we end up construction the api again, we can just recreate the api in the new thread.